src/main/java/com/synopsys/integration/alert/web/user/UserActions.java

```diff
@@ -88,6 +92,7 @@ public UserConfig createUser(UserConfig userConfig) throws AlertDatabaseConstrai
             authorizationUtility.updateUserRoles(userId, roleNames);
         }
         userModel = userAccessor.getUser(userId).orElse(userModel);
+        userSystemValidator.validateSysadminUser();
```

Comment:
I'm not sure we need to call this on a GET. I think after update should be enough.